heimlig/Cargo.lock

```diff
@@ -519,14 +519,15 @@ dependencies = [
 
 [[package]]
 name = "ed25519-dalek"
-version = "2.0.0"
+version = "2.1.1"
```

Comment:
Does it make sense to enforce these versions in the `Cargo.toml`?

Reply:
Good point. Done.